octoprint_v8theme/static/js/v8theme.js

```diff
@@ -1,7 +1,157 @@
 $(function() {
     function V8ThemeViewModel(parameters) {
         var self = this;
-        self.temp = parameters[0]
+        self.temperature = parameters[0];
+        self.terminal = parameters[1];
+        
+        var Colors = ["#fbc08d", "#fde0c8", "#ffb6a6", "#ffdbd4"];
```

Comment:
Capitalized var names are usually reserved for class names.  If you intended it to be a constant, you can use all caps, but lowercase is fine.